apps/mobile/src/navigation/AuthRootNavigator.js

```diff
@@ -4,8 +4,8 @@ import { createStackNavigator } from '@react-navigation/stack'
 import Intercom from '@intercom/intercom-react-native'
 import { LogLevel, OneSignal } from 'react-native-onesignal'
 import { gql, useMutation, useQuery, useSubscription } from 'urql'
-import i18n from '../../i18n'
 import mixpanel from 'services/mixpanel'
+import { useTranslation } from 'react-i18next'
```

Comment:
Sorry, this was just a "while I'm in here" tidy-up. Of no relevance to this PR otherwise.